### PR TITLE
Code Clean Up

### PR DESCRIPTION
**Pull request by GravityWolfNotAmused**
As I was looking around the codebase I noticed some inconsistencies in the way some things are written, not logically, but syntactically speaking.

The most common has to be how constructors are called. I have changed all that I found to new() as the variable type can be inferred. The second is between using the using directive or specifying the namespace on the variable type. In some instances this can cause ambiguity; however, the ones removed did not possess this ambiguity.

*Removed question for issue submission*

### CODE REVIEW

**Review by AMastryukov — APPROVED** [review]

**Review by mzegar — APPROVED** [review]
LGTM, thanks.

**Comment by GravityWolfNotAmused** [comment]
@mzegar You are welcome, I am going to do more prowling, and cleaning up after dinner. 
I made another commit to up one issue with formating that wasn't to the standard.

File: code/settings/Settings.cs [hunk 1]
```diff
@@ -115,17 +115,17 @@ public partial class SettingFunctions
 
         public static string GetJSON<T>(T settings, bool compressed = false) where T : Settings
```
Comment (Alf21) [inline code/settings/Settings.cs:116]:
```suggestion
        public static string GetJSON(T settings, bool compressed = false) where T : Settings
```

**Review by Alf21 — COMMENTED** [review]
GJ, ty! ❤️

**Comment by GravityWolfNotAmused** [comment]
@Alf21 I have committed your suggestion. I guess it went right past my eyes. 👍

**Review by Alf21 — APPROVED** [review]

**Comment by GravityWolfNotAmused** [comment]
@Alf21 There is an issue with the changes you asked for

**Comment by GravityWolfNotAmused** [comment]
@Alf21 We need that generic as we are passing a variable of type T.

**Review by Alf21 — APPROVED** [review]

**Pull request merged**